tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -269,6 +269,47 @@ class CompletionSuite extends BaseCompletionSuite {
            |synchronized[T0](x$1: T0): T0
            |toString(): String
            |""".stripMargin,
+      "3.2" ->
```

Comment:
```suggestion
      ">=3.2.0-RC1-bin-20220504-26f9e77-NIGHTLY" ->
```

Reply:
Actually, we might fix it in the compiler, so I will hold off for now.